packages/web/src/server.tsx

```diff
@@ -25,6 +27,28 @@ app.get('/robots.txt', (_, res) =>
   })
 );
 
+function getScripts(statsStr: string) {
+  let stats: Record<string, string[]>;
+  const assets: { js: string[]; css: string[] } = { js: [], css: [] };
+  try {
+    stats = JSON.parse(statsStr).assetsByChunkName as Record<string, string[]>;
```

Comment:
I was wondering if we can use `webpack` typings for additional type guarding in case something changes inside webpack

Reply:
Probably not (or at least I would not count on it). I think if something changes inside webpack we'll have much more interesting things other than updating stats format.